src/Backend/Core/Engine/Header.php

```diff
@@ -98,6 +98,17 @@ private function addCoreJs(): void
         $this->addJS('jquery/jquery.backend.js', 'Core', true, false, true, Priority::core());
         $this->addJS('utils.js', 'Core', true, false, true, Priority::core());
         $this->addJS('backend.js', 'Core', true, false, true, Priority::core());
+
+        $this->addJS('/js/vendors/select2/select2.full.js', 'Core', true, true, true, Priority::core());
+        $select2InterfaceLanguage = BL::getInterfaceLanguage();
+        if ($select2InterfaceLanguage === 'zh') {
+            $select2InterfaceLanguage = 'zh-TW';
```

Comment:
Maybe add a comment explaining what this does?

Reply:
fixed in 16a0f61b8